src/PostValidationHookInterface.php

```diff
@@ -5,11 +5,19 @@
 namespace Yiisoft\Validator;
 
 /**
- * Allows implementing post-validation processing in the data set object itself.
- *
- * @link https://github.com/yiisoft/form/blob/d2333f9a0a77f6dcb00db1ab8ee95ec4426ea133/src/FormModel.php#L202
+ * An optional interface for data set ({@see DataSetInterface}) to implement (used as a replacement). It provides
```

Comment:
Used as a replacement of what?

Reply:
`DatasetInterface` + `PostValidationHookInterface` - addition, wrong.
`DataSetInterface` -> `PostValidationHookInterface` - replacement, right.`

Reply:
@samdark suggested to remove it completely.